src/Schema/Event.ts

```diff
@@ -126,17 +160,117 @@ export interface SuccessfullyLoggedIn {
+ *    context_module: "auctionsRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Sale
+ *    destination_screen_owner_id: "5e95b37a2fdcb20012a0e082"
+ *    destination_screen_owner_slug: "forum-auctions-colour-theory-4"
+ *    horizontal_slide_position: 3
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedAuctionGroup {
+  action: ActionType.tappedAuctionGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  horizontal_slide_position?: number
+  module_height?: "single" | "double"
+  type?: "thumbnail" | "header" | "stub"
+}
+
+/**
+ * A user taps a grouping of collections on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedCollectionGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedCollectionGroup"
+ *    context_module: "collectionRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Collection
+ *    destination_screen_owner_slug: "limited-edition-prints-trending-artists"
+ *    horizontal_slide_position: 2
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedCollectionGroup {
+  action: ActionType.tappedCollectionGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  horizontal_slide_position?: number
+  module_height?: "single" | "double"
+  type: "thumbnail" | "header" | "stub"
+}
+
+/**
+ * A user taps a grouping of fairs on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedFairGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedFairGroup"
+ *    context_module: "fairRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Fair
+ *    destination_screen_owner_id: "5e726bd22524980012caafb0"
+ *    destination_screen_owner_slug: "arteba-special-edition"
+ *    horizontal_slide_position: 2
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedFairGroup {
+  action: ActionType.tappedFairGroup
   context_module: ContextModule
   context_screen_owner_type: ScreenOwnerType
   context_screen_owner_id?: string
   context_screen_owner_slug?: string
   destination_screen_owner_type: ScreenOwnerType
   destination_screen_owner_id?: string
   destination_screen_owner_slug?: string
-  freshness?: number
   horizontal_slide_position?: number
   module_height?: "single" | "double"
-  type: "button" | "thumbnail" | "header" | "stub"
+  type: "thumbnail" | "header" | "stub"
```

Comment:
What's a stub?

Reply:
good question! it's the info underneath the thumbnail on an artist/auction/collection card, for example: 
<img width="283" alt="Screen Shot 2020-04-24 at 10 27 53 AM" src="https://user-images.githubusercontent.com/26879583/80223588-79dab400-8616-11ea-8a91-cc320b3b0d2d.png">

We could call it something different, but I think 'stub' has been used in the past. @samchieng is there an official name for it?

Reply:
I've been calling it `Card Info` but I think Stub makes sense too if we've used that before in other places

Reply:
Sorry I never followed up on this but tapping on the stub vs. the thumbnail take you to the same place right? Is it important to us to know which one visitors tapped on?

Reply:
Each `type` can direct the user to a different destination within the same rail. Examples:

On the auctions rail, the header goes to the auctions home page, but thumbnails and stubs go to individual sale pages.

On the artist rails, headers aren't tappable but the thumbnails and stubs go to the artist page.

On artwork rails (specifically the 'based on other artist' rails), the header goes to an artist page, there are no stubs (presently), and the thumbnails go to artwork pages.

In short, I think it's important to know which was tapped.

Reply:
I think header is definitely important but it just seems like thumbnail vs. stub are the same thing. The reason I'm asking is because it's confusing to me which one to use for the collections rail! Will I need to specify two events for the collections rail? One for the thumbnail and one for the stub?

Reply:
i see what you're saying - in this case, i think it's ok to use `thumbnail` for everything. there are designs where the thumbnail and the stub are separately tappable, but in the event they are not, i would use `thumbnail` to describe the tap.

Reply:
Perfect, got it!